org.trelby.Trelby.yaml

```diff
@@ -0,0 +1,40 @@
+app-id: org.trelby.Trelby
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: trelby
+finish-args:
+  - --socket=cups
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --talk-name=org.freedesktop.FileManager1
+  - --filesystem=home
+  - --filesystem=/tmp
```

Comment:
Why is `/tmp` needed?

Reply:
Because when you click file > print via PDF, it stores a temporary copy under /tmp, opens your PDF reader for viewing and then you have the choice to do what you wish with that open file.

Reply:
is that all handled in this app or you mean a different app for viewing ?

Reply:
> is that all handled in this app or you mean a different app for viewing ?

If my understanding of the source code is correct, it doesn't print to the printer like GTK provides. It instead saves the PDF as a new file, puts it in /tmp, then then requires a PDF viewer to open said file. That is why there is a setting for selecting your PDF reader. It requires another app, so I used xdg-open to let the user select a PDF reader. Once it's open, you can then choose what you wish to do such as skim through it or use the built in printing options in the PDF viewer to do the actual printing be it to a file outside of tmp so it is saved permanent or straight to the printer itself as the user gets to decide.

Reply:
/tmp is a bit weird location to save. Why not save in `~/Documents` ? Why does it need to be temporary if the user needs to use different apps to view it

Reply:
Yes something other than `/tmp` would be better as this permission leaks whatever is in host's `/tmp`. You can use `~/.var/cache/$FLATPAK_ID/tmp` by setting `--env=TMPDIR=/var/tmp`, the only issue is that, it's not tmpfs like host's `/tmp`

Reply:
> the only issue is that, it's not tmpfs like host's `/tmp`
What kind of consequences or side effects would happen as a result?

Reply:
tmpfs means things will be cleaned up on restart. Nowadays most distros mount `/tmp` as tmpfs. Whereas that location is persistent storage. There's no cleanup on restart.

Reply:
> tmpfs means things will be cleaned up on restart. Nowadays most distros mount `/tmp` as tmpfs. Whereas that location is persistent storage. There's no cleanup on restart.

Going through the python source code, it uses the built in python3 temporary file creation functions, so it was always intended to be a temporary file with print menu option. I changed the tmp to use from /tmp (host tmp) to /var/tmp and the python program explicitly deletes it upon closing trelby. Any other changes I should make?